pkg/util/crypto.go

```diff
@@ -0,0 +1,76 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package util
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"math/big"
+	"time"
+)
+
+const (
+	DefaultRSABits            = 2048
+	DefaultValidDays          = 3650
```

Comment:
Maybe add a comment: for now the ClusterControlPlane API doesn't support rotating certificate. We set long valid time  to avoid certificate expiration.

Reply:
Added.